tests/unit/amo/pages/TestSearchPage.js

```diff
@@ -171,15 +171,22 @@ describe(__filename, () => {
     sinon.assert.callCount(fakeDispatch, 1);
   });
 
-  it('does not dispatch a server redirect when `platform` is not "all"', () => {
+  it('dispatches a server redirect when `platform` is set', () => {
```

Comment:
This test case is now _very_ similar to the one right above so maybe we don't need both?

Reply:
I debated removing the other, but it proves we don't do anything special with `all`, so that's a bit different.

Reply:
it does not really matter since both test cases check the exact same thing, it could be `all`, `whatever` or `lolilol`, there is nothing that checks the actual value in the code.

Reply:
I know, but there used to be :) If I just test `whatever`, that doesn't prove anything for `all`. It's technically the same coverage, but since `all` used to be special, I thought it'd best to prove it also redirects with no params in that case.

Reply:
we really don't need two test cases that are the same. In addition, if we had to test different values, we'd use `it.each`.

Reply:
6fe5b036112cbe73f056b81215b33faae1bf27f3